packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -89,7 +98,7 @@ const EDiscovery = WebexPlugin.extend({
    * @param {int} size - Number of records to retrieve
    * @returns {Promise<ResponseEntity<Array<ReportRecord>>>} Http Response containing a list of report records
    */
-  getReports({offset, size}) {
+  getReports({offset, size} = {}) {
```

Comment:
Added defaults here, both for testing purposes and as a general practice we're trying to adopt to avoid breaking changes to our consumers.